lightningd/lightningd.c

```diff
@@ -133,6 +135,28 @@ static void test_daemons(const struct lightningd *ld)
 	}
 	tal_free(ctx);
 }
+/* Check if all the daemons exist in the specified directory. */
+static bool has_all_daemons(const char* daemon_dir)
+{
+	size_t i;
+	bool retval;
+	const tal_t *ctx = tal_tmpctx(NULL);
```

Comment:
It makes me nervous to name tmpctxs "ctx"; when I see "tmpctx" I know exactly what's going on...

Reply:
`test_daemons` used `ctx`, so I copied it, but I shall modify this.